### DISCUSSION TITLE
Installation failed with "unable to extract selected files"

### DISCUSSION THREAD

**Discussion by FrzMtrsprt** [ppy/osu#18756] [Q&A] [answered]
When I click install.exe downloaded from the latest release, an error dialog (as follows) pops out. 
Running the installer as admin doesn't solve the issue, nor does a system reboot.
The `%localappdata%\SquirrelClowdTemp\` folder isn't created after running the installer, so I can't provide any logs.

System info:
Windows 11 (Beta 22621.160)
Apple Macbook Pro 12,1 (13 inch, early 2015)
(I can't think of any other info to provide that might help)

![image](https://user-images.githubusercontent.com/15852717/174434612-7f381f58-5f5b-4b5f-b664-67312c8b1ccb.png)

**Comment by integerrr** [comment, marked as answer]
Go ahead and bump #18369

**Reply by FrzMtrsprt** [reply]
Oh I didn't realise there were unicode characters in my username, that must be the problem. Thanks.
